modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,328 @@
+
+| Find files a|
+
+* Input parameters are now:
+ ** `Query` (JSON)
+ ** `Sort` (JSON)
+* Return type is now a JSON list.
+
+| Get file content a|
+
+* Input parameter is now `File id` (JSON).
+* Return type is now a JSON Result object that contains the file content and attributes.
+
+| Insert document | Return type is now a JSON object with the _id object of the created document.
+
+| Insert documents | Return type is now a Bulk Operation Result, which contains a JSON that lists each created record and its status.
+
+| List collections | Return type is now a List<String> that contains the names of the collections.
+
+| List indices | Operation is now called List indexes.
+
+| Map reduce objects a|
+
+* Operation is now called Map reduce.
+* Return type is now a JSON that represents the output specified in the reduce function.
+
+| Remove documents a|
+
+* Input parameters are now:
+ ** `Collection name` (String)
+ ** `Query` (JSON)
+* Return type is now a Long with the count result.
+
+| Remove files | Input parameter is now `File id` (JSON).
+
+| Update documents a|
+
+* Input parameters are now:
+ ** `Collection name` (String)
+ ** `Query` (JSON)
+ ** `Content to update` (JSON)
+ ** `Multiple update` (Boolean: moved to the Advanced tab)
+ ** `Upsert` (Boolean)
+* Return type is now a JSON that contains the following structure:
+** Int Matched
+** Int Modified
+** String Upserted Id
+
+|===
+
+== Changed Operations Metadata
+
+MongoDB 5.x generated operations metadata when the user provided a set of documents per collection from which to take the attributes. MongoDB 6.0 generates metadata automatically, based on the latest document in each collection.
+
+=== Metadata in MongoDB v6.0
+
+[%header,cols="34%,33%,33%"]
+|===
+|Operation Name	| Input Metadata	|Output Metadata
+
+| Insert Document
+|Document
+| Resolved dynamically based on the selected value of the collection parameter. The connector adds the latest document of the given collection and uses the document's structure as input/output metadata.
+
+| Insert Documents
+|Document
+|Resolved dynamically based on the selected value of the collection parameter. The connector adds the latest document of the given collection and uses the document's structure as input/output metadata.
+
+| Update Documents
+| N/A
+a| JSON that contains the following structure:
+
+* Int Matched
+* Int Modified
+* JSON upsertedId
+
+| Remove Documents
+| Query: JSON Object
+| N/A
+
+| Count Documents
+| Query: JSON Object
+| N/A
+
+| Find Documents
+| Query: JSON Object
+| Resolved dynamically based on the selected value of the collection parameter. The connector uses the structure of the first document in the given collection.
+
+| Create File
+| N/A
+a| A GridFS JSON object with the following structure:
+*Reviewers, should this say "with the following structure" or "with the following attributes, like the text for Find Files? Can we use the same terminology for all structures?"*
```

Comment:
yes, the structure is similar to Find Files operation: a JSON object with inner attributes. We can use "with the following attributes" here